Gulpfile.mjs

```diff
@@ -312,7 +314,7 @@ function buildRollup(packages, targetBrowsers) {
                 "./packages/babel-helper-create-regexp-features-plugin",
                 "regexpu-core",
                 "regenerate-unicode-properties"
-              ) + "/../**",
+              ) + "/**/*.js",
```

Comment:
Without the explicit `.js` extension it was trying to parse the LICENSE file as JS.